src/NetAppFiles/NetAppFiles/help/Approve-AzNetAppFilesReplication.md

```diff
@@ -231,3 +231,13 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 ## NOTES
 
 ## RELATED LINKS
+
+[Inititialize-AzNetAppFilesReplication](./Approve-AzNetAppFilesReplication.md)
```

Comment:
Why the link of Inititialize-AzNetAppFilesReplication points to Approve-AzNetAppFilesReplication.md